content/en/blog/contributor-contest/index.md

```diff
@@ -0,0 +1,133 @@
+tags = ["KubeEdge", "kubeedge", "edge computing", "kubernetes edge computing", "K8S edge orchestration", "edge computing platform", "KubeCon", "cloud native"]
+categories = ["General", "Announcements"]
+summary = "To encourage open source community members participation in KubeEdge project, this contest is been conducted. KubeEdge maintainers welcome community members, individuals, organizations & students for this contest. The event will start on 20th April 2019 and end on 23rd May 2019"
+
+# Projects (optional).
+#   Associate this post with one or more of your projects.
+#   Simply enter your project's folder or file name without extension.
+#   E.g. `projects = ["deep-learning"]` references 
+#   `content/project/deep-learning/index.md`.
+#   Otherwise, set `projects = []`.
+# projects = ["internal-project"]
+
+# Featured image
+# To use, add an image named `featured.jpg/png` to your page's folder. 
+[image]
+  # Caption (optional)
+  caption = ""
+
+  # Focal point (optional)
+  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
+  #focal_point = "Center"
++++
+
+  
+
+## **How to participate ?**
+
+{{% alert note %}}
+*Category 1*
+
+1. Raise pull request (PR) either for feature development / test code development (may be unit test code, edge module test code or end to end test code) in repos [kubeedge](https://github.com/kubeedge/kubeedge) / [beehive](https://github.com/kubeedge/beehive) / [viaduct](https://github.com/kubeedge/viaduct) / [website](https://github.com/kubeedge/website).
+2. Identify defects, raise issues in respective repos [kubeedge](https://github.com/kubeedge/kubeedge) / [beehive](https://github.com/kubeedge/beehive) / [viaduct](https://github.com/kubeedge/viaduct) / [website](https://github.com/kubeedge/website).
+3. Re-solving existing issues in repos [kubeedge](https://github.com/kubeedge/kubeedge) / [beehive](https://github.com/kubeedge/beehive) / [viaduct](https://github.com/kubeedge/viaduct) / [website](https://github.com/kubeedge/website).
+4. Share requirements by creating issues in repo [kubeedge](https://github.com/kubeedge/kubeedge).
+{{% /alert %}}
+
+{{% alert note %}}
+*Category 2*
+
+Writing blogs about KubeEdge either in the [website](https://github.com/kubeedge/website) (on PR approved & merged, this gets published in kubeedge.io website blog) or in other popular technical blogging site.
+
+Submit your blog details  [here](https://docs.google.com/forms/d/e/1FAIpQLSd9luGaHWt5jABEQjtr11CgWLVw7Px6tJf0TIjidbl3UL_Izg/viewform).
+
+{{% /alert %}}
+
+{{% alert note %}}
+*Category 3*
+
+Create your own sample applications and demo examples to illustrate possible use case(s) of using KubeEdge in repo [examples](https://github.com/kubeedge/examples).
+{{% /alert %}}
+
+
+## **How the winners are selected**
+
+{{% alert note %}}
+In case of Category 1
+
+KubeEdge maintainers will review the code submitted with PR(s) for feature / test case development or issue fix. 
```

Comment:
Any code contributions will follow the [contribution flow](https://github.com/kubeedge/kubeedge/blob/master/CONTRIBUTING.md) to get accepted.

Reply:
Done